htdocs/js/ui/notebook_protection_controller.js

```diff
@@ -94,6 +97,24 @@ define(['angular'], function(angular) {
 
         //NOTEBOOK TAB METHODS
         /////////////////////////////////////////
+        $scope.getUsers = function() {
+            return $q(function(resolve, reject) {
+                rcloud.get_users()
+                .then(function(data) {
+
+                    $scope.$evalAsync(function() {
+                        $scope.allUsers = _.map(data, function(user) { return { text : user, value : user }; });
+                    });
+
+                    //timeout to ensure evalAsync has finished
+                    $timeout(function() {
+                        logger.clear();
+                        resolve();
```

Comment:
Hmm, is async-call-then-timeout-resolve really the right pattern? Seems like we should just resolve once the asynchronous call is done.